tests/api/test_aws.py

```diff
@@ -3,6 +3,7 @@
 from unittest.mock import MagicMock, patch
 
 import boto3
+import botocore
```

Comment:
Do you need to import `botocore` here?

Reply:
Good catch, I was using `ClientError` or whatever is called from `botocore` in this test file but then I changed it. Thanks, I'll remove 👍